### DISCUSSION TITLE
Still haven't received github student package benefits, it's been 6 days

### DISCUSSION THREAD

**Discussion by ophixyle** [community/community#146169] [GitHub Education]
### Still haven't received github student package benefits, it's been 6 days

<img width="233" alt="Screenshot 2024-12-03 184438" src="https://github.com/user-attachments/assets/2f2e0b48-a790-4b68-8ca9-03a79844d2b7">
I'm a bit worried, as usually it's supposed to arrive in 3-4 days or earlier. What could the issue be and what should I do? Thanks :)

**Comment by davevad93** [comment]
Hi @ophixyle ,

Don't worry it's normal, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ophixyle was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
